pkg/apiserver/rest/usecase/authentication.go

```diff
@@ -288,6 +267,84 @@ func (a *authenticationUsecaseImpl) GetDexConfig(ctx context.Context) (*apisv1.D
 	}, nil
 }
 
+func (a *authenticationUsecaseImpl) UpdateDexConfig(ctx context.Context) error {
+	dexApp, config, err := getDexConfig(ctx, a.kubeClient)
+	if err != nil {
+		return err
+	}
+	// TODO: use the real connection data
+	test := []interface{}{
+		map[string]string{
+			"id":   "mock",
+			"name": "Example",
+			"type": "mockCallback",
+		},
+	}
+	config.Connectors = test
```

Comment:
Why add a test connector config?

Reply:
I hope this can be merged first for @zzxwill , after the user creates a dex config, the `UpdateDexConfig` will be called.